src/main/java/cz/smarteon/loxone/message/EncryptedCommand.java

```diff
@@ -102,10 +102,6 @@ public static EncryptedCommand<Token> refreshToken(final String tokenHash, final
     }
 
     private static String encodeUrl(String toEncode) {
-        try {
```

Comment:
could you kindly move changes in this file to separate commit?

Reply:
Added in separate pull request